tooling/lib/version.js

```diff
@@ -197,12 +197,12 @@ async function checkLastCommit() {
 }
 
 /**
- * Determines if there are any breaking changes between HEAD and upstream/master
+ * Determines if there are any breaking changes between HEAD and upstream/beta
```

Comment:
Should we change `beta` to `next`

Reply:
I think this isn't effective in the next branch. It was last updated in `next` branch 10 months ago and it still says `upstream/master`

So, I believe it is not relevant anymore